WooCommerce/Classes/POS/Presentation/Card Present Payments/Reader Messages/PointOfSaleCardPresentPaymentDisplayReaderMessageMessageViewModel.swift

```diff
@@ -1,5 +1,17 @@
 import Foundation
 
 struct PointOfSaleCardPresentPaymentDisplayReaderMessageMessageViewModel {
+    let title: String = Localization.title
     let message: String
+    let imageName = String.posProcessingPaymentImageName
```

Comment:
Could you use this from PointOfSaleAssets instead of the String extension please? You may need to merge trunk to see it... but this won't work after the merge otherwise, as the string extension is gone.

Reply:
Of course. Yes, this is an outdated branch.